### PR TITLE
Added setType and getType to MonoSynth

### LINKED ISSUE #663: Changing osc type in MonoSynth
Is there a reason why you can't change the oscillator type of the monosynth directly? 
like through the constructor or a setType function

### PR DESCRIPTION
**Pull request by KevinGrajeda**
Resolves #663 
Added `setType(type)` and `getType()` to monosynth
Added documentation and tests

### CODE REVIEW

**Comment by KevinGrajeda** [comment]
@davepagurek I would like a review, thanks👍

File: src/monosynth.js [hunk 1]
```diff
@@ -298,6 +298,29 @@ class MonoSynth extends AudioVoice {
...
+   *  @method  setType
+   *  @for p5.MonoSynth
+   *  @param {String} type 'sine', 'triangle', 'sawtooth' or 'square'.
+   */
+  setType(type) {
+    this.oscillator.setType(type);
+  }
+
+  /**
+   *  Returns  current type of internal oscillator eg. 'sine', 'triangle', 'sawtooth' or 'square'.
```
Comment (davepagurek) [inline src/monosynth.js:313]: super minor but there are two spaces after "returns," mind taking one out?

**Review by davepagurek — COMMENTED** [review]
Looks great! One very tiny formatting thing and otherwise good to go. Thanks for adding tests too!

**Comment by KevinGrajeda** [comment]
It's done 👍

**Pull request merged**